src/webapp/src/components/Footer.js

```diff
@@ -4,41 +4,32 @@ import { withStyles } from '@material-ui/core/styles';
 import { Checkbox, FormGroup } from "@material-ui/core";
 import { ipcRenderer } from 'electron';
 
-const AUTO_LAUNCH_EXPLANATION = "Run Rookout's Desktop App on machine startup";
+const AUTO_LAUNCH_EXPLANATION = "Run Dynatrace Live Debugging Desktop App on machine startup";
 
 const styles = {
     root: {
-        color: blueGrey[500],
+        color: '#A9AAF2 !important',
         '&$checked': {
-            color: blueGrey[500],
+            color: '#A9AAF2 !important',
         },
     },
     checked: {},
 }
 
-export const Footer = ({ classes, ...props }) => {
+export const Footer = ({ classes }) => {
     const [autoLaunchEnabled, setAutoLaunchEnabled] = useState(false);
-    const [exceptionManagerEnabled, setExceptionManagerEnabled] = useState(false);
 
     useEffect(() => {
         ipcRenderer.on("auto-launch-is-enabled-changed", (event, isEnabled) => {
             setAutoLaunchEnabled(isEnabled);
         });
-        ipcRenderer.on("exception-manager-enabled-changed", (event, isEnabled) => {
```

Comment:
2 things:
1. Is the event fully cleaned from the codebase?
2. Make sure store is cleared from the "should collect" field, otherwise data might be collected automatically when it is reimplemented without the users being able to consent

Reply:
I removed the event wherever I could find it

As for the setting itself, obviously we'll bring back the checkbox if there's a new implementation of data collection, so I don't see why it's necessary to check the setting now or at all, it'll just create a confusing bit of code that's completely unneeded.

Reply:
@ron-rookout Ok. Just make sure the setting has a new name in the store to avoid using the old setting's value